cmd/aznum2words-webapp/aznum2words-webapp.go

```diff
@@ -0,0 +1,97 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"github.com/alexliesenfeld/health"
+	"github.com/egasimov/aznum2words/cmd/aznum2words-webapp/api"
+	"github.com/egasimov/aznum2words/cmd/aznum2words-webapp/config"
+	"github.com/jessevdk/go-flags"
+	"github.com/joho/godotenv"
+	"github.com/labstack/echo/v4"
+	echomiddleware "github.com/labstack/echo/v4/middleware"
+	"github.com/labstack/gommon/log"
+	"time"
+)
+
+var LoggerConfig = echomiddleware.LoggerConfig{
+	Format: `{"time":"${time_rfc3339_nano}","x-correlation-id":"${header:x-correlation-id}","remote_ip":"${remote_ip}",` +
+		`"host":"${host}","method":"${method}","uri":"${uri}","user_agent":"${user_agent}",` +
+		`"status":${status},"error":"${error}","latency":${latency},"latency_human":"${latency_human}"}` + "\n",
+	CustomTimeFormat: "2006-01-02 15:04:05.00000",
+}
+
+func main() {
+	var err error
+	_, err = flags.Parse(&config.Opts)
+	if err != nil {
+		panic(err)
+	}
+
+	initEnvVars()
+	config.LoadConfig()
+
+	// Create an instance of our handler which satisfies the generated interface
+	var azNum2WordsApi = new(api.AzNum2WordsApi)
+
+	// This is how you set up a basic Echo router
+	e := echo.New()
+	// Log all requests
```

Comment:
```
	e.Use(echomiddleware.RequestIDWithConfig(
		echomiddleware.RequestIDConfig{
			TargetHeader: echo.HeaderXCorrelationID,
	}))

```

it will generate value for given header, and set for response

Reply:
Added